opentelemetry/src/sdk/trace/span_processor.rs

```diff
@@ -180,7 +189,7 @@ impl SpanProcessor for SimpleSpanProcessor {
 /// [`async-std`]: https://async.rs
 pub struct BatchSpanProcessor {
     message_sender: Mutex<mpsc::Sender<BatchMessage>>,
-    worker_handle: Option<Pin<Box<dyn Future<Output = ()> + Send + Sync>>>,
+    _worker_handle: Option<Pin<Box<dyn Future<Output = ()> + Send + Sync>>>,
```

Comment:
do we still need this reference?

Reply:
Nope I guess we don't really need it. Will remove